modules/gcs/main.tf

```diff
@@ -128,13 +128,43 @@ resource "google_storage_bucket_object" "objects" {
   }
 }
 
-resource "google_storage_bucket_iam_binding" "bindings" {
+resource "google_storage_bucket_iam_binding" "authoritative" {
   for_each = var.iam
   bucket   = google_storage_bucket.bucket.name
   role     = each.key
   members  = each.value
 }
 
+resource "google_storage_bucket_iam_binding" "bindings" {
```

Comment:
Miren, can you use a `iam.tf` file for IAM resources, like in all other modules?

Reply:
done